data/json/npcs/robofac/robofac_intercom/robofac_intercom_trades.json

```diff
@@ -232,5 +232,235 @@
+      "effect": [
+        { "npc_add_var": "dialogue_intercom_accepted_yrax_robot", "value": "yes" }
+      ],
+      "sentinel": "yrax_trade"
+    }
+  },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_TRIAKIS",
+    "type": "talk_topic",
+    "dynamic_line": "We can look at this but it appears to miss quite alot to it.",
+    "speaker_effect": {
+      "effect": [
+        { "npc_add_var": "dialogue_intercom_accepted_yrax_robot", "value": "yes" }
+      ],
+      "sentinel": "yrax_trade"
+    }
+  },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_GOLDEN_MONOLITH",
+    "type": "talk_topic",
+    "dynamic_line": "That is quite a lot of broken metal, let's hope R&D can at least salvage materials.",
+    "speaker_effect": {
+      "effect": [
+        { "npc_add_var": "dialogue_intercom_accepted_yrax_robot", "value": "yes" }
+      ],
+      "sentinel": "yrax_trade"
+    }
+  },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_DELTA",
+    "type": "talk_topic",
+    "dynamic_line": "You don't need to pick up all glossy scrap that you find.",
+    "speaker_effect": {
+      "effect": [
+        { "npc_add_var": "dialogue_intercom_accepted_yrax_robot", "value": "yes" }
+      ],
+      "sentinel": "yrax_trade"
+    }
+  },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_QUADRAPHRACT_LEG",
+    "type": "talk_topic",
+    "dynamic_line": "We witnessed it as a part of larger construct.",
+    "speaker_effect": {
+      "effect": [
+        { "npc_add_var": "dialogue_intercom_accepted_yrax_robot", "value": "yes" }
+      ],
+      "sentinel": "yrax_trade"
+    }
+  },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_QUADRAPHRACT",
+    "type": "talk_topic",
+    "dynamic_line": "I'm impressed you managed to drag it inside.  We gonna need to fix lift after this one.",
+    "speaker_effect": {
+      "effect": [
+        { "npc_add_var": "dialogue_intercom_accepted_yrax_robot", "value": "yes" }
+      ],
+      "sentinel": "yrax_trade"
+    }
+  },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_SELL_BOT_YRAX_TRIFACET",
+    "type": "talk_topic",
+    "dynamic_line": "I belive we already have similar model already but is nice to have more, it will allow us be more active in our research.",
+    "speaker_effect": {
+      "effect": [
+        { "npc_add_var": "dialogue_intercom_accepted_yrax_robot", "value": "yes" }
+      ],
+      "sentinel": "yrax_trade"
+    }
+  },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_SELL_BOT_YRAX_TRIAKIS",
+    "type": "talk_topic",
+    "dynamic_line": "Oh what an interesting piece R&D will have a field day with it.",
+    "speaker_effect": {
+      "effect": [
+        { "npc_add_var": "dialogue_intercom_accepted_yrax_robot", "value": "yes" }
+      ],
+      "sentinel": "yrax_trade"
+    }
+  },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_SELL_BOT_GOLDEN_MONOLITH",
+    "type": "talk_topic",
+    "dynamic_line": "I assume you were trying to figure out limits of hm12.  I guess they are right here.",
```

Comment:
Probably `the hm12`,, but I think this works too in this case.

Reply:
not sure about that either

Reply:
I think:  "figure out limits of hm12" = "figure out the limits of the class of rifles named hm12", "figure out limits of the hm12" = either "figure out the limits of the platonic hm12" or "figure out the limits of your hm12". In this case, while technically different messages, there isn't any particular difference.